2021-04-25 14:51:51,745 INFO: Microblog startup [in /home/ec2-user/environment/lihkg/app/__init__.py:74]
2021-04-25 14:54:15,059 INFO: Microblog startup [in /home/ec2-user/environment/lihkg/app/__init__.py:74]
2021-04-25 14:59:12,092 INFO: Microblog startup [in /home/ec2-user/environment/lihkg/app/__init__.py:74]
2021-04-25 15:00:26,039 INFO: Microblog startup [in /home/ec2-user/environment/lihkg/app/__init__.py:74]
2021-04-25 15:21:42,881 INFO: Microblog startup [in /home/ec2-user/environment/lihkg/app/__init__.py:74]
2021-04-25 15:32:59,496 INFO: Microblog startup [in /home/ec2-user/environment/lihkg/app/__init__.py:74]
2021-04-25 16:32:52,888 INFO: Microblog startup [in /home/ec2-user/environment/lihkg/app/__init__.py:74]
2021-04-26 05:20:44,970 INFO: Microblog startup [in /home/ec2-user/environment/lihkg/app/__init__.py:74]
2021-04-26 05:25:55,537 INFO: Microblog startup [in /home/ec2-user/environment/lihkg/app/__init__.py:74]
2021-04-26 06:03:16,366 INFO: Microblog startup [in /home/ec2-user/environment/lihkg/app/__init__.py:74]
2021-04-26 06:03:51,378 INFO: Microblog startup [in /home/ec2-user/environment/lihkg/app/__init__.py:74]
2021-04-26 06:12:11,366 INFO: Microblog startup [in /home/ec2-user/environment/lihkg/app/__init__.py:74]
2021-04-26 06:17:01,810 INFO: Microblog startup [in /home/ec2-user/environment/lihkg/app/__init__.py:74]
2021-04-26 06:26:49,647 INFO: Microblog startup [in /home/ec2-user/environment/lihkg/app/__init__.py:74]
2021-04-29 21:20:52,610 INFO: Microblog startup [in /home/ec2-user/environment/lihkg/app/__init__.py:74]
2021-04-29 21:23:55,078 INFO: Microblog startup [in /home/ec2-user/environment/lihkg/app/__init__.py:74]
2021-04-29 22:01:35,112 INFO: Microblog startup [in /home/ec2-user/environment/lihkg/app/__init__.py:74]
2021-04-29 22:06:42,095 INFO: Microblog startup [in /home/ec2-user/environment/lihkg/app/__init__.py:74]
2021-04-29 22:18:57,803 INFO: Microblog startup [in /home/ec2-user/environment/lihkg/app/__init__.py:74]
2021-04-30 00:10:08,942 INFO: Microblog startup [in /home/ec2-user/environment/lihkg/app/__init__.py:74]
2021-04-30 00:39:57,097 INFO: Microblog startup [in /home/ec2-user/environment/lihkg/app/__init__.py:74]
2021-04-30 01:58:16,397 INFO: Microblog startup [in /home/ec2-user/environment/lihkg/app/__init__.py:74]
2021-04-30 02:04:13,290 INFO: Microblog startup [in /home/ec2-user/environment/lihkg/app/__init__.py:74]
2021-04-30 02:06:10,053 INFO: Microblog startup [in /home/ec2-user/environment/lihkg/app/__init__.py:74]
2021-04-30 02:08:07,993 INFO: Microblog startup [in /home/ec2-user/environment/lihkg/app/__init__.py:74]
2021-04-30 02:08:53,438 INFO: Microblog startup [in /home/ec2-user/environment/lihkg/app/__init__.py:74]
2021-04-30 02:12:05,929 INFO: Microblog startup [in /home/ec2-user/environment/lihkg/app/__init__.py:74]
2021-04-30 02:37:55,975 INFO: Microblog startup [in /home/ec2-user/environment/lihkg/app/__init__.py:74]
2021-04-30 02:38:54,204 INFO: Microblog startup [in /home/ec2-user/environment/lihkg/app/__init__.py:74]
2021-04-30 02:38:57,339 INFO: Microblog startup [in /home/ec2-user/environment/lihkg/app/__init__.py:74]
2021-04-30 03:03:08,110 INFO: Microblog startup [in /home/ec2-user/environment/lihkg/app/__init__.py:74]
2021-04-30 04:13:34,464 INFO: Microblog startup [in /home/ec2-user/environment/lihkg/app/__init__.py:74]
2021-04-30 04:14:40,393 INFO: Microblog startup [in /home/ec2-user/environment/lihkg/app/__init__.py:74]
2021-04-30 04:15:08,435 INFO: Microblog startup [in /home/ec2-user/environment/lihkg/app/__init__.py:74]
2021-04-30 05:10:24,466 INFO: Microblog startup [in /home/ec2-user/environment/lihkg/app/__init__.py:74]
2021-05-01 07:35:58,505 INFO: Microblog startup [in /home/ec2-user/environment/lihkg/app/__init__.py:74]
2021-05-01 07:36:29,994 INFO: Microblog startup [in /home/ec2-user/environment/lihkg/app/__init__.py:74]
2021-05-01 08:15:10,715 INFO: Microblog startup [in /home/ec2-user/environment/lihkg/app/__init__.py:74]
2021-05-01 08:17:00,110 INFO: Microblog startup [in /home/ec2-user/environment/lihkg/app/__init__.py:74]
2021-05-01 08:17:10,869 INFO: Microblog startup [in /home/ec2-user/environment/lihkg/app/__init__.py:74]
2021-05-01 08:17:25,092 INFO: Microblog startup [in /home/ec2-user/environment/lihkg/app/__init__.py:74]
2021-05-01 08:27:35,664 INFO: Microblog startup [in /home/ec2-user/environment/lihkg/app/__init__.py:74]
2021-05-01 08:29:02,638 INFO: Microblog startup [in /home/ec2-user/environment/lihkg/app/__init__.py:74]
2021-05-01 08:29:06,945 INFO: Microblog startup [in /home/ec2-user/environment/lihkg/app/__init__.py:74]
2021-05-01 08:29:59,461 INFO: Microblog startup [in /home/ec2-user/environment/lihkg/app/__init__.py:74]
2021-05-01 08:30:57,219 INFO: Microblog startup [in /home/ec2-user/environment/lihkg/app/__init__.py:74]
2021-05-01 08:31:16,746 INFO: Microblog startup [in /home/ec2-user/environment/lihkg/app/__init__.py:74]
2021-05-01 08:37:38,990 INFO: Microblog startup [in /home/ec2-user/environment/lihkg/app/__init__.py:74]
2021-05-01 08:38:26,231 INFO: Microblog startup [in /home/ec2-user/environment/lihkg/app/__init__.py:74]
2021-05-01 08:48:28,277 INFO: Microblog startup [in /home/ec2-user/environment/lihkg/app/__init__.py:74]
2021-05-01 08:48:42,694 INFO: Microblog startup [in /home/ec2-user/environment/lihkg/app/__init__.py:74]
2021-05-01 08:50:59,039 INFO: Microblog startup [in /home/ec2-user/environment/lihkg/app/__init__.py:74]
2021-05-01 08:51:50,948 INFO: Microblog startup [in /home/ec2-user/environment/lihkg/app/__init__.py:74]
2021-05-01 08:52:04,020 INFO: Microblog startup [in /home/ec2-user/environment/lihkg/app/__init__.py:74]
2021-05-01 08:52:12,250 INFO: Microblog startup [in /home/ec2-user/environment/lihkg/app/__init__.py:74]
2021-05-01 08:56:53,234 INFO: Microblog startup [in /home/ec2-user/environment/lihkg/app/__init__.py:74]
2021-05-01 08:58:50,304 ERROR: Exception on /casual [GET] [in /home/ec2-user/.local/lib/python3.7/site-packages/flask/app.py:1761]
Traceback (most recent call last):
  File "/home/ec2-user/.local/lib/python3.7/site-packages/sqlalchemy/engine/base.py", line 1193, in _execute_context
    context)
  File "/home/ec2-user/.local/lib/python3.7/site-packages/sqlalchemy/engine/default.py", line 508, in do_execute
    cursor.execute(statement, parameters)
sqlite3.OperationalError: no such table: post

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/ec2-user/.local/lib/python3.7/site-packages/flask/app.py", line 2292, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/ec2-user/.local/lib/python3.7/site-packages/flask/app.py", line 1815, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/ec2-user/.local/lib/python3.7/site-packages/flask/app.py", line 1718, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/ec2-user/.local/lib/python3.7/site-packages/flask/_compat.py", line 35, in reraise
    raise value
  File "/home/ec2-user/.local/lib/python3.7/site-packages/flask/app.py", line 1813, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/ec2-user/.local/lib/python3.7/site-packages/flask/app.py", line 1799, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/ec2-user/environment/lihkg/app/main/routes.py", line 47, in casual
    page, current_app.config['POSTS_PER_PAGE'], False)
  File "/home/ec2-user/.local/lib/python3.7/site-packages/flask_sqlalchemy/__init__.py", line 491, in paginate
    items = self.limit(per_page).offset((page - 1) * per_page).all()
  File "/home/ec2-user/.local/lib/python3.7/site-packages/sqlalchemy/orm/query.py", line 2783, in all
    return list(self)
  File "/home/ec2-user/.local/lib/python3.7/site-packages/sqlalchemy/orm/query.py", line 2935, in __iter__
    return self._execute_and_instances(context)
  File "/home/ec2-user/.local/lib/python3.7/site-packages/sqlalchemy/orm/query.py", line 2958, in _execute_and_instances
    result = conn.execute(querycontext.statement, self._params)
  File "/home/ec2-user/.local/lib/python3.7/site-packages/sqlalchemy/engine/base.py", line 948, in execute
    return meth(self, multiparams, params)
  File "/home/ec2-user/.local/lib/python3.7/site-packages/sqlalchemy/sql/elements.py", line 269, in _execute_on_connection
    return connection._execute_clauseelement(self, multiparams, params)
  File "/home/ec2-user/.local/lib/python3.7/site-packages/sqlalchemy/engine/base.py", line 1060, in _execute_clauseelement
    compiled_sql, distilled_params
  File "/home/ec2-user/.local/lib/python3.7/site-packages/sqlalchemy/engine/base.py", line 1200, in _execute_context
    context)
  File "/home/ec2-user/.local/lib/python3.7/site-packages/sqlalchemy/engine/base.py", line 1413, in _handle_dbapi_exception
    exc_info
  File "/home/ec2-user/.local/lib/python3.7/site-packages/sqlalchemy/util/compat.py", line 203, in raise_from_cause
    reraise(type(exception), exception, tb=exc_tb, cause=cause)
  File "/home/ec2-user/.local/lib/python3.7/site-packages/sqlalchemy/util/compat.py", line 186, in reraise
    raise value.with_traceback(tb)
  File "/home/ec2-user/.local/lib/python3.7/site-packages/sqlalchemy/engine/base.py", line 1193, in _execute_context
    context)
  File "/home/ec2-user/.local/lib/python3.7/site-packages/sqlalchemy/engine/default.py", line 508, in do_execute
    cursor.execute(statement, parameters)
sqlalchemy.exc.OperationalError: (sqlite3.OperationalError) no such table: post [SQL: 'SELECT post.id AS post_id, post.title AS post_title, post.body AS post_body, post.timestamp AS post_timestamp, post.category_id AS post_category_id, post.user_id AS post_user_id \nFROM post ORDER BY post.timestamp DESC\n LIMIT ? OFFSET ?'] [parameters: (25, 0)] (Background on this error at: http://sqlalche.me/e/e3q8)
